Data/ApplicationDbContext.cs

```diff
@@ -22,6 +22,20 @@ public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options)
         public DbSet<ActivityType> ActivityTypes { get; set; }
         public DbSet<Module> Modules { get; set; }
         public DbSet<Course> Courses { get; set; }
-        public DbSet<SystemUserCourse> UserCourses { get; set; }
+        public DbSet<SystemUserCourse> SystemUserCourses { get; set; }
+
+        protected override void OnModelCreating(ModelBuilder builder)
+        {
+            base.OnModelCreating(builder);
+
+            builder.Entity<SystemUserCourse>()
+                .HasKey(k => new
+                {
+                    k.SystemUserId,
+                    k.CourseId
+                });
+        }
+
+        public DbSet<LexiconLMS.Core.ViewModels.SystemUserViewModel> SystemUserViewModel { get; set; }
```

Comment:
This shouldn't be in the database, should it?

Reply:
woopsie